src/Bicep.MSBuild.E2eTests/examples/multiple/multiple.proj

```diff
@@ -3,7 +3,9 @@
 -->
 <Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <!-- Ensure the package supports multi-targeting -->
+    <TargetFrameworks>net5;net472</TargetFrameworks>
+    <LanguageTargets>$(MSBuildThisFileDirectory)\multitarget-mitigation.targets</LanguageTargets> <!-- https://github.com/microsoft/MSBuildSdks/issues/155 -->
```

Comment:
>LanguageTargets [](http://example.com/codeflow?start=5&length=15)

I think we'll need to update the msbuild docs to mention this workaround after this PR gets merged.